api/validators.py

```diff
@@ -32,7 +32,7 @@ def validate_data(data, schema_json, schema_type, verb, optional=False):
     validator(data, verb)
 
 def _validate_json(json_data, schema, resolver):
-    jsonschema.validate(json_data, schema, resolver=resolver)
+    jsonschema.validate(json_data, schema, resolver=resolver, format_checker=jsonschema.FormatChecker())
```

Comment:
Wow. For those who, like me, were wondering why this is necessary, it turns out `format` is optional:

> JSON Schema impls are not required to implement this part of the specification, and many of them do not.

No big deal, but something to keep in mind if we a) change libraries, or b) want to validate in another language.